apidoc/Titanium/Contacts/Person.yml

```diff
@@ -160,15 +178,31 @@ properties:
         `string` type value.
         
         The `service` key value may be one of `AIM`, `Facebook`, `GaduGadu`, `GoogleTalk`, `ICQ`, 
-        `MSN`, `QQ`, `Skype`, or `Yahoo`.
+        `Jabber`, `MSN`, `QQ`, `Skype`, or `Yahoo`.
         
         On BlackBerry, the following apply: `aim`, `aliwangwang`, `bbmPin`, `googleTalk`, `icq`,
-        `irc`, `jabber`, `msLcs`, `msn`, `qq`, `sametime`, `skype`, `yahooMessenger`, `yahooMessenger`
+        `irc`, `jabber`, `msLcs`, `msn`, `qq`, `sametime`, `skype`, `yahooMessenger`
 
 
     type: Dictionary
     platforms: [android, iphone, ipad, blackberry]
     
+  - name: socialProfile
+    summary: |
+        Social profile information of the person. Multi-value.
+    description: |
+        The format of the dictionary accepted by this property is as follows:
+        
+        * Keys: any of `home`, `work` and/or `other`.
```

Comment:
I think the keys are incorrect. Shouldn't they be `facebookProfile`, `flickrProfile` etc?